clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -1203,6 +1203,7 @@ void ConvertCIRToLLVMPass::runOnOperation() {
                            mlir::func::FuncDialect>();
 
   getOperation()->removeAttr("cir.sob");
+  getOperation()->removeAttr("cir.lang");
```

Comment:
It's fine to remove it for now, but I think longer term we might want to do this in `applyPartialConversion` for ModuleOp where we can convert the attribute to debug attribute.